js/bundle/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const B1 = 'b1';
+export const B2 = 'b2';
+export const DEFAULT_VERSION = B2;
+export const APPROVED_VERSIONS = [B1, B2];
```

Comment:
```suggestion
export const APPROVED_VERSIONS = [B1, B2] as const;
```

Marking this with `as const` can lead to better type inference, since the type is currently `string[]`, but with `as const` it would literally be `['b1', 'b2']`.

Reply:
Done